app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -42,6 +43,9 @@ class Settings private constructor(context: Context) {
                     false); */
             false
 
+    fun shouldShowOnboarding(): Boolean =
+            !preferences.getBoolean(OnboardingActivity.ONBOARD_PREF, false)
```

Comment:
Let's set this default value to true - worst case they never see it, rather than worst case they see it *every* time.

Reply:
We can't do this – we rely on the default being false in order to show it the first time.